lib/widgets/transaction_item.dart

```diff
@@ -91,12 +92,16 @@ class TransactionsItemState extends State<TransactionsItem> {
   }
 
   int sendValue(GeneralTransaction vti) {
-    if (vti.txnType == TransactionType.value_transfer) {
-      bool isInternalTx =
-          externalAddresses.contains(vti.vtt!.outputs[0].pkh.address) ||
-              internalAddresses.contains(vti.vtt!.outputs[0].pkh.address) ||
-              singleAddressAccount?.address == vti.vtt!.outputs[0].pkh.address;
-      return isInternalTx ? vti.fee : vti.vtt!.outputs[0].value.toInt();
+    if (vti.type == TransactionType.value_transfer) {
```

Comment:
What do you think about refactoring this function to remove unnecessary nesting?

```dart
  int sendValue(GeneralTransaction vti) {
    if (vti.type != TransactionType.value_transfer || vti.vtt!.outputs.length < 0) {
      return 0;
    }

    bool isInternalTx = externalAddresses
            .contains(vti.vtt!.outputs[0].pkh.address) ||
        internalAddresses.contains(vti.vtt!.outputs[0].pkh.address) ||
        singleAddressAccount?.address == vti.vtt!.outputs[0].pkh.address;
    return isInternalTx ? vti.fee : vti.vtt!.outputs[0].value.toInt();
  }
```